extra-dev/packages/coq-freespec-ffi/coq-freespec-ffi.dev/opam

```diff
@@ -22,10 +22,10 @@ build: [
 depends: [
   "ocaml"
   "dune" {>= "2.5"}
-  "coq" {>= "8.12" & < "8.13~"}
+  "coq" {>= "8.12" & < "8.14~" | = "dev"}
```

Comment:
It's very difficult to be compatible with COq 8.12/8.13/dev but *not* 8.14/8.15. Since this is a `dev` package, how about we just set a lower bound:
```suggestion
  "coq" {>= "8.12"}
```